FreeRTOS/FreeRTOS-Kernel
# Update mpu_wrappers_v2.c

Description
-----------
This PR updates mpu_wrappers_v2.c to add value to uninitialized variables .

Test Steps
-----------
Run [CORTEX_MPU_M3_MPS2_QEMU_GCC](https://github.com/FreeRTOS/FreeRTOS/tree/main/FreeRTOS/Demo/CORTEX_MPU_M3_MPS2_QEMU_GCC)

Checklist:
----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
